benchmark/src/main/scala/code/winitzki/benchmark/Benchmarks9.scala

```diff
@@ -107,36 +107,81 @@ object Benchmarks9 {
     result
   }
 
+  val counterMultiplier = 10
+
```

Comment:
Should not all benchmark_9* methods  expect a tp instead of threads parameter consistent with other benchmarks and avoiding cost of doing a ThreadPool creation inside. To me it's mostly a style/consistency issue.